modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -966,6 +964,9 @@ private void _testSystemObjectRelationshipOneToMany() throws Exception {
 		_addObjectRelationshipSystemObjectDefinition();
 	}
 
+	private static final Pattern _manyToManyRelationshipTableNamePattern =
+		Pattern.compile("R_[A-Z][0-9][A-Z][0-9]$");
```

Comment:
@thallescezar please rename it to `_manyToManyObjectRelationshipTableNamePattern` (or just `Pattern _pattern`)